### PR TITLE
[FSDE-26322] Created Tabs web component

### PR DESCRIPTION
**Pull request by janibolkvadze**

### CODE REVIEW

File: src/custom-elements/src/components/dropdown/dropdown.tsx [hunk 1]
```diff
@@ -148,6 +163,14 @@ export class Dropdown {
...
   }
 
+  displayBlock() {
+    this._dropdownMenuElement.style.display = 'block';
+  }
+
+  displayNone() {
+    this._dropdownMenuElement.style.display = 'none';
+  }
+
```
Comment (SergioQCL) [inline src/custom-elements/src/components/dropdown/dropdown.tsx:173]: These could be merged into one single method and a parameter. Thoughts?

File: src/custom-elements/src/components/tabs/tabs.tsx [hunk 2]
```diff
@@ -0,0 +1,485 @@
...
+  private lisArrayLength = 0;
+  private liSizes = {};
+  private seeMoreTriggerAnchorElement: HTMLAnchorElement;
+  private seeMoreTriggerElement: HTMLLIElement;
+  private seeMoreTriggerElementWidth = 0;
+  private ulElement: HTMLUListElement;
+  private uuid4: string = uuid4();
+  private seeMoreTriggerId: string = `see-more-trigger-${this.uuid4}`;
+  private slidingBorderLeft: any;
+  private slidingBorderWidth: any;
```
Comment (SergioQCL) [inline src/custom-elements/src/components/tabs/tabs.tsx:96]: Shouldn't these be strings?

**Review by SergioQCL — COMMENTED** [review]
Just a couple of minor details

File: src/custom-elements/src/components/dropdown/dropdown.tsx (hunk 1 above)
Comment (janibolkvadze) [reply src/custom-elements/src/components/dropdown/dropdown.tsx:173]: agree, I suggest `setDisplay()` which would accept `'block' | 'none'`

File: src/custom-elements/src/components/tabs/tabs.tsx (hunk 2 above)
Comment (janibolkvadze) [reply src/custom-elements/src/components/tabs/tabs.tsx:96]: Numbers as the values are later concatenated `${slidingBorderLeft}px`

**Review by dani-cl-madrid — APPROVED** [review]

**Review by jllr — APPROVED** [review]

**Review by SergioQCL — APPROVED** [review]
BMB

**Pull request merged**